test/e2e/tests/confirmations/transactions/contract-interaction-redesign.spec.ts

```diff
@@ -238,10 +252,10 @@ describe('Confirmation Redesign Contract Interaction Component', function () {
           smartContract,
           title: this.test?.fullTitle(),
         },
-        async ({ driver, contractRegistry }) => {
-          const contractAddress = await contractRegistry.getContractAddress(
-            smartContract,
-          );
+        async ({ driver, contractRegistry }: TestSuiteArguments) => {
+          const contractAddress = await (
```

Comment:
Minor, is this worth a small function to avoid the duplication?

Reply:
Added in c326fdb8d3